### PR TITLE
removing system python from mac os x walkt through

### PR DESCRIPTION
**Pull request by atarkowska**
This PR update Mac OS X walkthrough
- removes system python support
- clean up

### CODE REVIEW

**Comment by jburel** [comment]
`environemnt` typo

File: omero/sysadmins/unix/server-install-homebrew.txt [hunk 1]
```diff
@@ -100,20 +100,7 @@ Check that Python is working and is version 2.7::
...
-If using Homebrew Python, this will be on the default module path.
-
-.. note::
-
-    If you have a local :file:`.bash_profile` file, it will override
-    your :file:`.profile` configuration file.
-
-
-Independently of the chosen Python, you can set up and use `virtual
+To create isolated Python environemnt you can set up and use `virtual
```
Comment (mtbc) [inline omero/sysadmins/unix/server-install-homebrew.txt:102]: "To create _an_ isolated ..."

**Comment by joshmoore** [comment]
Looks good, @aleksandra-tarkowska (barring @mtbc's suggestion). But for clarification before we merge, why is there no ~~`homebrew/python`~~ `homebrew/science` tapping in https://github.com/ome/homebrew-alt/blob/master/.travis.yml ? @sbesson ? Is there some unification that needs happening?

**Comment by sbesson** [comment]
I am confused. `homebrew-science` is certainly tapped on https://github.com/ome/homebrew-alt/blob/master/.travis.yml#L13 for the HDF5 dependency.
If the comment was about `homebrew/python`, this was also my main question about this PR. Are we going the `homebrew/python` route to install some of the Python dependencies (`numpy`, `scipy`) as opposed to pip? If so, we might want to modify the install script and test it before merging this PR.

**Comment by joshmoore** [comment]
Sorry, yes, I meant `homebrew/python`

**Comment by atarkowska** [comment]
@joshmoore @sbesson we had that commands duplicated in the main install page https://github.com/openmicroscopy/ome-documentation/pull/1333/files#diff-7a80e8edf4f934dad7d77ed5642814a4L74. If that is not needed, I am happy to remove python.

**Comment by sbesson** [comment]
For refrence, the `homebrew/python` tap was introduced in https://github.com/openmicroscopy/ome-documentation/pull/1212. As far as I know, `homebrew/python` is not mandatory and is not tested and I would remove it from the walkthrough minimally. About leaving it on the main UNIX installation page, this may be reviewed as part of the deployment work of OMERO 5.2.1 /cc @jburel

**Comment by atarkowska** [comment]
@sbesson done

**Comment by hflynn** [comment]
Is this ready to merge now?

**Comment by sbesson** [comment]
This works for me. LGTM as other changes will be coming for this page.

**Pull request merged**